docs/lessons/dance-party.md

```diff
@@ -0,0 +1,511 @@
+# Dance Party
+
+Click on the keys as they appear on the screen! When your player collides with the blocks you will get points. Try to beat your high score and dance away!
+
+![Game view in simulator](/static/lessons/dance-party.png)
```

Comment:
Might make the header a gif instead of an image - the lessons very visual and a short gameplay gif here feels a bit more enticing to me